public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -225,20 +268,28 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
             <EuiTitle size="l">
               <h1>
                 {detector && detector.name}{' '}
-                {detector && detector.enabled ? (
+                {detector &&
+                detector.enabled &&
+                detector.curState === DETECTOR_STATE.RUNNING ? (
                   <EuiHealth color="success">
                     Running since{' '}
                     {detector.enabledTime
-                      ? moment(detector.enabledTime).format('MM/DD/YY h:mm a')
+                      ? moment(detector.enabledTime).format('MM/DD/YY h:mm A')
                       : '?'}
                   </EuiHealth>
+                ) : detector.enabled &&
+                  detector.curState === DETECTOR_STATE.INIT ? (
+                  <EuiHealth color="primary">Initializing</EuiHealth>
+                ) : detector.curState === DETECTOR_STATE.INIT_FAILURE ||
+                  detector.curState === DETECTOR_STATE.UNEXPECTED_FAILURE ? (
+                  <EuiHealth color="danger">Initialization failure</EuiHealth>
                 ) : (
                   <EuiHealth color="subdued">
                     {detector.featureAttributes &&
                     detector.featureAttributes.length
                       ? detector.disabledTime
                         ? `Stopped at ${moment(detector.disabledTime).format(
-                            'MM/DD/YY h:mm a'
+                            'MM/DD/YY h:mm A'
                           )}`
                         : 'Detector is not started'
                       : 'Feature required to start the detector'}
```

Comment:
can use the `DETECTOR_STATE_COLORS` enum here for consistency w/ detector list page. can call `stateToColorMap(detector.curState)` to retrieve the color

Reply:
Sure, seems this needs some testing and tune. Will create an issue to track this. After we merged code, will work on this.

Reply:
Yeah, sounds good

Reply:
Track with this issue: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/56